src/main/java/ch/poole/openinghoursparser/OpeningHoursParseException.java

```diff
@@ -43,7 +46,7 @@ public int getColumn() {
     @Override
     public String getMessage() {
```

Comment:
Shouldn't we be using _getLocalizedMessage_ to return translated messages and just leave _getMessage_ as is? I admit to never using the former though too, so maybe this is the wrong place for Java purity :-).r